vortexor/src/stake_updater.rs

```diff
@@ -0,0 +1,98 @@
+//! Module responsible for updating the staked key map.
+//! Adapted from jito-replayer code.
+
+use {
+    crate::load_balancer::RpcLoadBalancer,
+    log::warn,
+    solana_client::client_error,
+    solana_sdk::pubkey::Pubkey,
+    solana_streamer::streamer::StakedNodes,
+    std::{
+        collections::HashMap,
+        str::FromStr,
+        sync::{
+            atomic::{AtomicBool, Ordering},
+            Arc, RwLock,
+        },
+        thread::{self, sleep, Builder, JoinHandle},
+        time::{Duration, Instant},
+    },
+};
+
+const PK_TO_STAKE_REFRESH_DURATION: Duration = Duration::from_secs(5);
```

Comment:
what is `PK`? `pubkey`? If that's the case, I think just call this refresh stake duration - it's clear stake is associated with a pubkey